src/Models/Scopes/Product/WithProductAttributesScope.php

```diff
@@ -26,7 +26,8 @@ public function apply(Builder $builder, Model $model)
             ->addSelect($builder->getQuery()->from . '.entity_id AS id')
             ->addSelect($builder->getQuery()->from . '.sku')
             ->addSelect($builder->getQuery()->from . '.visibility')
-            ->addSelect($builder->getQuery()->from . '.type_id AS type');
+            ->addSelect($builder->getQuery()->from . '.type_id AS type')
```

Comment:
In that case i would suggest using the qualifyColumn functions, and selecting a column `AS` using the built in way in addSelect as it contains additional safeties
```suggestion
            ->addSelect(['id' => $model->getQualifiedKeyName()])
            ->addSelect($model->qualifyColumn('sku'))
            ->addSelect($model->qualifyColumn('visibility'))
            ->addSelect(['type' => $model->qualifyColumn('type_id')])
```

Reply:
It's still harder to read?

Reply:
It's master so here we can make bigger changes; maybe it's a good idea to get rid of the aliases? And just go with:
```php
$builder->addSelect([
    $model->getQualifiedKeyName(),
    $model->qualifyColumn('sku'),
    $model->qualifyColumn('visibility'),
    $model->qualifyColumn('type_id'),
    $model->getQualifiedCreatedAtColumn(),
]);
```

The aliases are just some simple replaces, it would be nice if we can track them down within the Rapidez repo's and where it's used also replace it with the qualify column. When we're going to update a project we'll notice it pretty fast; so no need to keep it backwards compatible.

Reply:
That also works for me, just gotta find and eliminate all the `id` and `type` usages which I'm sure are numerous.